fancybox/fancybox.php

```diff
@@ -28,27 +28,43 @@ function fancybox_footer(App $a, string &$str)
 	DI::page()->registerFooterScript(__DIR__ . '/asset/fancybox/fancybox.config.js');
 }
 
-function fancybox_render(App $a, array &$b)
-{
-	$matches = [];
-	$pattern = '#<div class="body-attach">.*?</div>#s';
-	$gallery = 'gallery';
-	if (array_key_exists('item', $b)) {
-		$item = $b['item'];
-		if (array_key_exists('uri-id', $item)) {
-			$gallery = $gallery . '-' . $item['uri-id'];
-		}
-	}
-	$html = $b['html'];
-	while (preg_match($pattern, $html, $matches, PREG_OFFSET_CAPTURE)) {
-		if (is_array($matches)) {
-			$matches = $matches[0];
-		}
-		$part     = $matches[0];
-		$replaced = str_replace('<a href', '<a data-fancybox="' . $gallery . '" href', $part);
-		$replaced = str_replace('<div class="body-attach"', '<div class="body-attach done"', $replaced);
-		$html     = str_replace($part, $replaced, $html);
+function fancybox_render($a, array &$b){
```

Comment:
Why did you remove the type hint of the first parameter?
```suggestion
function fancybox_render(App $a, array &$b){
```

Reply:
This was because of test code. I forget to restore it.